go/caddy/api/api.go

```diff
@@ -58,12 +61,32 @@ func buildHandler(ctx context.Context) (APIHandler, error) {
 
 	router.POST("/api/v1/nodes/fingerbank_communications", apiHandler.nodeFingerbankCommunications)
 
-	NewAdminApiAuditLog().AddToRouter(router)
-	NewAuthLog().AddToRouter(router)
-	NewDnsAuditLog().AddToRouter(router)
-	NewRadacctLog().AddToRouter(router)
-	NewRadiusAuditLog().AddToRouter(router)
-	NewWrix().AddToRouter(router)
+	done := false
+	var DB *gorm.DB
+	var err error
+	for !done {
```

Comment:
Shouldn't we limit that for a number of retry? like 2 or 3 max.
Then just exit and fail if not working

Reply:
We don't have to, since mysql is a must for most of the feature to be available.